redfish/openapi/dmtf/openapi-reduced.yaml

```diff
@@ -76,6 +76,50 @@ paths:
               schema:
                 $ref: '#/components/schemas/RedfishError'
           description: Error condition
+  /redfish/v1/odata:
```

Comment:
move this to the end of the file to keep the same order as in the DMTF openapi spec